browser/src/index.tsx

```diff
@@ -14,7 +14,13 @@ const defaultSettings: ISettings = window['settings'];
 const store = configureStore({ 
     settings: defaultSettings, 
     graph: {}, 
-    vscode: { theme: document.body.className, locale: window['locale'], configuration: window['configuration'] } });
+    vscode: { 
+        theme: document.body.className,
+        locale: window['locale'],
+        configuration: window['configuration'],
+        api: window['vscode']
```

Comment:
Lets not stick this into the store. Perhaps a light weight wrapper as decribed earlier